3_managing_roles/README.md

```diff
@@ -1,3 +1,108 @@
 # Managing Roles
 
-A collection of examples regarding managing roles.
+A collection of examples regarding managing Roles.
+
+## Migrating to Access Rules
+
+To increase flexibility when managing thousands of Resources, Role Grants have
+been deprecated in favor of Access Rules, which allow you to grant access based
+on Resource Tags and Type. Role Grants will be removed from the Terraform
+provider by March 15, 2022.
```

Comment:
Before we publish we should reaffirm this date. Or better yet, if it's really that soon, not mention it. I'll ask about this.